docs/activities.md

```diff
@@ -0,0 +1,96 @@
+# Sprint Activities List
+
+## Most popular
+
+### Wine tasting - 35€
+
+Dive deeper into the unique style of how wine is produced on Lanzarote.
+
+Choose four of your favourite styles from a portfolio of wines.
+
+A bite of artisanal cheese and charcuterie will accompany your personalized tasting. The experience takes about 60 minutes.
+
+### Vulcano hike - free
+
+A classic of Ocean Sprint. We leave the venue for a 1.5h hike to the nearest vulcano with great views over the nearby tows and coastline.
+
+Beginner-friendly.
+
+### Intro to surfing - 30€
+
+Pickup by vans at the sprint venue and leave for Famara, the surfing mecca of Lanzarote. Two hours in the water with rented surfboard, wetsuits and surf coaching. Back by dinner.
+
+Beginner-friendly.
+
+### Bonfire Night in the Desert - free
+
+A night under the stars with a bonfire in the middle of a vulcano. 
+
+We'll organize the transportation and each participants brings their own food&drinks.
+
+## Recommended
+
+
+### Sailing - TBA
+
+Enjoy the calm sea and scenic views during a few hour sailing excursion at sunset
+with drinks and tapas.
+
+### Swimming - free
+
+Go for a morning swim 5min away from the venue.
```

Comment:
Meet at venue at 8AM, walk down to the beach for a quick swim, be back by 9AM breakfast time.